svgdigitizer/entrypoint.py

```diff
@@ -58,6 +58,12 @@ def cli():
     help="Detect non-orthogonal skewed axes going through the markers instead of assuming that axes are perfectly horizontal and vertical.",
 )
 
+si_option = click.option(
+    "--si",
```

Comment:
```suggestion
    "--si-units",
```

`si` is very short and can be misunderstood for `-s -i`. At least that was what I read at first in the examples.

Reply:
agree